flink-formats/flink-json/src/test/java/org/apache/flink/formats/json/canal/CanalJsonSerDeSchemaTest.java

```diff
@@ -105,6 +112,34 @@ void testIgnoreParseErrors() throws Exception {
                                 "An error occurred while collecting data."));
     }
 
+    @Test
+    void testIgnoreParseErrorsLogsDebugMessage() throws Exception {
```

Comment:
could you add a test case for fail on missing as well please. This also comes out of `JsonParserToRowDataConverters` as a `JsonParseException`.